pkg/oc/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -97,24 +97,33 @@ func (h *Helper) DockerRoot() (string, error) {
 }
 
 // Version returns the Docker API version and whether it is a Red Hat distro version
-func (h *Helper) APIVersion() (string, bool, error) {
+func (h *Helper) APIVersion() (*types.Version, error) {
```

Comment:
Instead of having `IsRedHad` can't we just return here 3 values: version, isRH, error ?

Reply:
Usually both checks will be somehow related, so why not just one.

Reply:
it was like that before, I hated that we hide the red hat check in APIVersion

Reply:
> it was like that before, I hated that we hide the red hat check in APIVersion

I agree with this sentiment.  One function for one purpose.  It calls out weirdness and makes it very explicit what we're doing and why.